Loop/Managers/DeviceDataManager.swift

```diff
@@ -264,7 +269,7 @@ final class DeviceDataManager {
             self.cgmManager = pumpManager as? CGMManager
         }
 
-        statusExtensionManager = StatusExtensionDataManager(deviceDataManager: self)
+        statusExtensionManager = StatusExtensionDataManager(deviceDataManager: self, closedLoopStatusObservable: closedLoopStatusObservable)
 
         loopManager = LoopDataManager(
```

Comment:
The instantiation of these non-device related managers should probably be moved at some point to LoopAppManager, and then LoopAppManager can wire up the connections between them. Not needed in this PR; just noticing we still need to do this.

Reply:
I'll include this as a TODO in the code.